_about/groups/task-forces/wcag2ict.md

```diff
@@ -0,0 +1,99 @@
+---
+title: WCAG2ICT Task Force of the AG WG
+permalink: /about/groups/task-forces/wcag2ict/
+ref: /about/groups/task-forces/wcag2ict/
+lang: en
+github:
+  repository: w3c/wai-about-wai
+  path: '_about/groups/task-forces/wcag2ict.md'
+---
+
+{::nomarkdown}
+{% include box.html type="start" title="Summary" class="" %}
+{:/}
+
+The WCAG2ICT Task Force is a task force of the [Accessibility Guidelines Working Group (AG WG)](/about/groups/agwg/).
+
+Some information on this page is also shown on, and may be more current in, the [WCAG2ICT Task Force page](https://www.w3.org/groups/tf/wcag2ict/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2..3" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Page Contents" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Announcements
+
+[WCAG2ICT Draft for review](https://www.w3.org/TR/wcag2ict-22/)
+
+## Meetings and Communication
+
+The WCAG2ICT TF conducts its work using a variety of synchronous and asynchronous tools. Work is being done via teleconference, email and GitHub issues.
+
+- [Email lists](https://www.w3.org/WAI/GL/task-forces/wcag2ict/#email);
+- [wcag2ict GitHub repository](https://github.com/w3c/wcag2ict/) and [GitHub draft document](https://w3c.github.io/wcag2ict/);
+- IRC options
+
+- IRC discussion on the
+- [#wcag2ict](http://irc.w3.org/?channels=#wcag2ict) IRC channel, used largely for minute-taking;
+- Server: irc.w3.org; Port 6665, 6667, 21, or 994; Channel: #wcag2ict
+
+- [Web-Based Surveys (WBS)](https://www.w3.org/2002/09/wbs/55145/);
+- [Teleconferences of the task force](https://www.w3.org/WAI/GL/task-forces/wcag2ict/#teleconferences) (also see [meeting minutes](https://www.w3.org/WAI/GL/task-forces/wcag2ict/minutes));
+- Face to face meetings;
```

Comment:
i think we should keep link to minutes as-is for now until we decide what to do with the minutes tool.